docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -90,6 +91,15 @@ C:\> where ocaml
 C:\Program Files\ocpwin64\4.02.1+ocp1-msvc64-20160113\bin\ocaml.exe
 ```
 
+Openssl for Windows
```

Comment:
Standardize on the correct name casing for "OpenSSL"

Reply:
ok, will fix

Reply:
OK, will fix this in the document